### PR TITLE
[TECH] Adding internationalization

### PR DESCRIPTION
**Pull request by snaf-trip**
LinguiJS is installed, but you can't switch between languages.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "plugins": [
+    "macros"
+  ]
+}
```
Comment (mtalstykh-exadel) [inline .babelrc]: add a new line

File: .linguirc [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+   "locales": ["en", "ru"],
+   "catalogs": [{
+      "path": "src/locales/{locale}/messages",
+      "include": ["src"]
+   }],
+   "format": "minimal"
+}
```
Comment (mtalstykh-exadel) [inline .linguirc]: add a new line

File: src/App.js [hunk 3]
```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 import SimpleCounter from './store/store';
-
 import './App.scss';
 
+//for Internationalization
```
Comment (mtalstykh-exadel) [inline src/App.js]: the comment is not so useful

File: src/App.js [hunk 4]
```diff
@@ -1,8 +1,10 @@
...
+import { Trans } from '@lingui/macro';
+
 // importing the elements Material-UI
```
Comment (mtalstykh-exadel) [inline src/App.js]: same here

File: src/App.js [hunk 5]
```diff
@@ -29,10 +31,10 @@ function App() {
     <div className="App">
       {/* Adding the imported element */}
```
Comment (mtalstykh-exadel) [inline src/App.js]: and here

File: src/index.js [hunk 6]
```diff
@@ -4,14 +4,22 @@ import './index.scss';
...
+import { i18n } from '@lingui/core';
+import { I18nProvider } from '@lingui/react';
+//Start on eng
+import catalogEn from './locales/en/messages.js';
+i18n.load('en', catalogEn.messages);
+i18n.activate('en');
+//start on ru
+// import catalogRu from './locales/ru/messages.js';
+// i18n.load('ru', catalogRu.messages);
+// i18n.activate('ru');
```
Comment (mtalstykh-exadel) [inline src/index.js]: Why do we need the code if it is not used?

File: src/locales/en/messages.js [hunk 7]
```diff
@@ -0,0 +1 @@
+/*eslint-disable*/module.exports={messages:{"Example button Material-UI":"Example button Material-UI","Example №2 disabled":"Example №2 disabled"}};
```
Comment (mtalstykh-exadel) [inline src/locales/en/messages.js]: format it right way please

File: src/locales/ru/messages.js [hunk 8]
```diff
@@ -0,0 +1 @@
+/*eslint-disable*/module.exports={messages:{"Example button Material-UI":"Пример кнопки Material-UI","Example №2 disabled":"Пример №2 disabled"}};
```
Comment (mtalstykh-exadel) [inline src/locales/ru/messages.js]: format it right way please

**Review by tensenrey — DISMISSED** [review]

File: src/locales/en/messages.js [hunk 9]
```diff
@@ -0,0 +1,7 @@
+/*eslint-disable*/
```
Comment (mtalstykh-exadel) [inline src/locales/en/messages.js]: do we need it here?

File: src/locales/ru/messages.js [hunk 10]
```diff
@@ -0,0 +1,7 @@
+/*eslint-disable*/
```
Comment (mtalstykh-exadel) [inline src/locales/ru/messages.js]: same here

File: src/index.js [hunk 11]
```diff
@@ -3,15 +3,17 @@ import ReactDOM from 'react-dom';
 import './index.scss';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { i18n } from '@lingui/core';
+import { I18nProvider } from '@lingui/react';
+//Start on eng
+import catalogEn from './locales/en/messages.js';
+i18n.load('en', catalogEn.messages);
```
Comment (mtalstykh-exadel) [inline src/index.js:12]: we should make it dynamic based on useres language from localstorage or navigator.language

File: src/index.js [hunk 12]
```diff
@@ -3,15 +3,17 @@ import ReactDOM from 'react-dom';
...
+i18n.activate('en');
 
 ReactDOM.render(
-  <React.StrictMode>
```
Comment (mtalstykh-exadel) [inline src/index.js]: why did you delete it?

File: src/locales/en/messages.js (hunk 9 above)
Comment (mtalstykh-exadel) [reply src/locales/en/messages.js]: may be the file should be in gitignore because it seems like auto generated

File: src/locales/ru/messages.js (hunk 10 above)
Comment (mtalstykh-exadel) [reply src/locales/ru/messages.js]: may be the file should be in gitignore because it seems like auto generated

File: src/locales/ru/messages.js [hunk 13]
```diff
@@ -0,0 +1 @@
+
```
Comment (mtalstykh-exadel) [inline src/locales/ru/messages.js]: the file should be ignored

File: src/locales/en/messages.js [hunk 14]
```diff
@@ -0,0 +1 @@
+
```
Comment (mtalstykh-exadel) [inline src/locales/en/messages.js]: the file should be ignored

File: src/App.js [hunk 15]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import './App.scss';
 import Login from './pages/login/login';
 import { Switch, Route } from 'react-router-dom';
 import './App.scss';
```
Comment (mtalstykh-exadel) [inline src/App.js]: two App.scss...
